test/functional/apps/discover/esql/_esql_view.ts

```diff
@@ -199,7 +199,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const cell = await dataGrid.getCellElementExcludingControlColumns(0, 1);
         expect(await cell.getVisibleText()).to.be(' - ');
         expect(await dataGrid.getHeaders()).to.eql([
-          "Select columnPress the Enter key to interact with this cell's contents.", // contains screen reader help text
+          'Select column',
```

Comment:
Hi @cee-chen,

Sorry, I don't have a chance to review the PR fully this week. From a quick look, it seems like the row controls can get misaligned depending on the display settings:

<img width="1255" alt="Screenshot 2024-09-26 at 17 32 55" src="https://github.com/user-attachments/assets/d250d3d6-8508-4fa5-91eb-c4abcbe9037b">

Reply:
Will take a look at this, thanks for flagging it Julia!

Reply:
Fixed now with the latest commits:

<img width="412" alt="" src="https://github.com/user-attachments/assets/cd93b9df-bc6f-499a-99d1-ded2dfa4cc41">

<img width="305" alt="" src="https://github.com/user-attachments/assets/e70ea79f-ddb1-48d3-9397-8c943150ce26">

The EUI changes let me clean up a bunch of extra wrappers while also making the checkboxes/buttons bounce less when switching between single and custom! 🥳

Reply:
Great, thanks! 😍